versiondb/tsrocksdb/comparator.go

```diff
@@ -0,0 +1,52 @@
+package tsrocksdb
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/linxGnu/grocksdb"
+)
+
+// CreateTSComparator should be compatible with builtin timestamp comparator.
+func CreateTSComparator() *grocksdb.Comparator {
+	return grocksdb.NewComparatorWithTimestamp(
+		"leveldb.BytewiseComparator.u64ts", TimestampSize, compare, compareTS, compareWithoutTS,
```

Comment:
should we call different name instead it?

Reply:
I deliberate to use the same name as rocksdb builtin comparator, so the result database can still work with builtin tools like `ldb`/`sst_dump`.